pkg/commands/start_host_windows.go

```diff
@@ -14,36 +14,29 @@ limitations under the License.
 package commands
 
 import (
-	"os"
-	"os/signal"
-	"syscall"
-
 	"github.com/rancher-sandbox/rancher-desktop-host-resolver/pkg/dns"
-	"github.com/sirupsen/logrus"
+	"github.com/rancher-sandbox/rancher-desktop-host-resolver/pkg/vmsock"
+	log "github.com/sirupsen/logrus"
```

Comment:
Isn't this package normally imported as `logrus`? Why are we renaming it (making the use less clear)?

Reply:
I honestly think log is more clear than logrus, but I'm ok either way 🤷